addons/controllers/clusterbootstrap_controller.go

```diff
@@ -210,6 +213,24 @@ func (r *ClusterBootstrapReconciler) reconcileNormal(cluster *clusterapiv1beta1.
 		return ctrl.Result{}, err
 	}
 
+	_, err = r.createOrPatchResourcesForCorePackages(cluster, clusterBootstrap, remoteClient, log)
+	if err != nil {
+		return ctrl.Result{RequeueAfter: time.Second * 10}, err
+	}
+
+	_, err = r.createOrPatchResourcesForAdditionalPackages(cluster, clusterBootstrap, remoteClient, log)
+	if err != nil {
+		return ctrl.Result{Requeue: true}, err
+	}
+
+	return ctrl.Result{}, nil
+}
+
```

Comment:
createOrPatchResourcesForCorePackages and createOrPatchResourcesForAdditionalPackages had to be introduced to reduce the cyclomatic complexity or go lint complains: 
```clusterbootstrap_controller.go:179:1: cyclomatic complexity 17 of func `(*ClusterBootstrapReconciler).reconcileNormal` is high (> 15) (gocyclo)
func (r *ClusterBootstrapReconciler) reconcileNormal(cluster *clusterapiv1beta1.Cluster, log logr.Logger) (ctrl.Result, error) {
^

Reply:
Fixed  to RequeueAfter: constants.RequeueAfterDuration in both cases. The functions only return err != nil. Is up to the caller to modifie the reque  request